app/src/androidTest/java/com/pse2022/timefy/ui/dashboard/DashboardUiTest.kt

```diff
@@ -0,0 +1,196 @@
+
+@RunWith(AndroidJUnit4::class)
+class DashboardUiTest {
+
+    @get:Rule
+    val composeRule = createComposeRule()
+
+    private val server = MockWebServer()
+    lateinit var authManager: AuthManager
+    lateinit var eventService: EventService
+    lateinit var viewModel: DashboardViewModel
+
+    @Before
+    fun setUp() {
+
+        server.start(8000)
+
+        val baseURL = server.url("/").toString()
+
+        val okHttpClient = OkHttpClient
+            .Builder()
+            .build()
+
+        eventService = Retrofit.Builder()
+            .baseUrl(baseURL)
+            .addConverterFactory(GsonConverterFactory.create())
+            .client(okHttpClient)
+            .build().create(EventService::class.java)
+
+        authManager = MockAuthManager()
+
+        composeRule.setContent {
+            TimefyTheme() {
+                viewModel = viewModel(factory = DashboardViewModel.provideFactory(eventService, authManager))
+                DashboardView(viewModel, null)
+            }
+        }
+    }
+
+    @After
+    fun tearDown() {
+        server.shutdown()
+    }
+
+    @Test
+    fun testNewlyPlannedEvents() {
+        server.enqueue(
+            MockResponse()
+                .setResponseCode(200)
+                .setBody(
+                    """
+                    [{
+		"id": "UUID",
+		"title": "Event title",
+		"duration": 60,
+		"synced": false,
+		"startDate": 0,
+        "plannedAt": 0,
+		"visibility": "PUBLIC",
+		"numParticipants": 5,
+		"maxParticipants": 10,
+		"creator": {
+			"id": "UUID",
+			"displayName": "FirstName LastName#UID"
+		}
+	}
+]
+                    """
+                )
+        )
+        composeRule.onNodeWithTag(TestTags.TITLE_BAR_TITLE).assertTextEquals("Hallo,\nMax Mustermann")
+
+        composeRule.waitUntil {
+            viewModel.newlyPlannedItems.value.isNotEmpty()
+        }
+
+        composeRule.onNodeWithTag(TestTags.DASHBOARD_NEWLY_PLANNED_SECTION, useUnmergedTree = true).assertIsDisplayed().assertTextEquals("Neu festgelegte Events")
+        composeRule.onNodeWithTag(TestTags.DASHBOARD_NEWLY_PLANNED_ITEM, useUnmergedTree = true).assertIsDisplayed()
+        composeRule.onNodeWithTag(TestTags.DASHBOARD_NEWLY_PLANNED_ITEM_TITLE, useUnmergedTree = true).assertIsDisplayed().assertTextEquals("Event title")
+        composeRule.onNodeWithTag(TestTags.DASHBOARD_NEWLY_PLANNED_ITEM_DATE, useUnmergedTree = true).assertIsDisplayed().assertTextEquals("01.01.1970, 01:00 - 02:00 Uhr")
+        composeRule.onNodeWithTag(TestTags.DASHBOARD_NEWLY_PLANNED_ITEM_CREATOR, useUnmergedTree = true).assertIsDisplayed().assertTextEquals("Erstellt von FirstName LastName#UID")
+    }
+
+    @Test
+    fun testUpcomingEvents() {
+        server.enqueue(
+            MockResponse()
+                .setResponseCode(200)
+                .setBody(
+                    """
+                    [{
+		"id": "UUID",
+		"title": "Event title",
+		"duration": 60,
+		"synced": false,
+		"startDate": 0,
```

Comment:
```suggestion
		"startDate": 3600,
```
Same applies to this

Reply:
Dito.